### PR TITLE
PMM-10117 migrate QAN to monorepo

### PR DESCRIPTION
**Pull request by atymchuk**

### CODE REVIEW

File: pmm/pmm2-release-candidate.groovy [hunk 1]
```diff
@@ -42,7 +42,6 @@ void runPMM2OVFBuild(String SUBMODULES_GIT_BRANCH, String RELEASE_CANDIDATE) {
 def pmm_submodules() {
     return [
         "pmm",
-        "qan-api2",
```
Comment (PavelKhripkov) [inline pmm/pmm2-release-candidate.groovy:45]: What exactly happens when we remove this line? Don't we exclude the module from the build process?

File: pmm/pmm2-release.groovy [hunk 2]
```diff
@@ -504,7 +504,6 @@ ENDSSH
                         declare -A repos=(
                             ["percona-grafana"]="percona-platform/grafana"
                             ["percona-dashboards"]="percona/grafana-dashboards"
-                            ["percona-qan-api2"]="percona/qan-api2"
```
Comment (PavelKhripkov) [inline pmm/pmm2-release.groovy:507]: Not sure about what this line is responsible for. Looks like we're dropping a repo but don't provide any replacement.

File: pmm/pmm2-release-candidate.groovy (hunk 1 above)
Comment (atymchuk) [reply pmm/pmm2-release-candidate.groovy:45]: Yes, we do, but not the module, but the repo. It'll now be build from `percona/pmm` vs `percona/qan-api2`.

File: pmm/pmm2-release.groovy (hunk 2 above)
Comment (atymchuk) [reply pmm/pmm2-release.groovy:507]: It creates a tag on those repos. In this case it's already done on line 508.

**Pull request merged**